superset-frontend/src/dashboard/components/nativeFilters/utils.ts

```diff
@@ -94,6 +95,7 @@ export const getFormData = ({
     viz_type: filterType,
     type,
     dashboardId,
+    native_filter_id: id,
```

Comment:
Calling this `id` seemed a tad terse. I'm not sure if this should be snake or camel case.